backend/core/detection.py

```diff
@@ -296,68 +304,63 @@ def handle_bgp_update(self, message: Dict) -> NoReturn:
                 monitor_event['timestamp'] = datetime(
                     *map(int, re.findall('\d+', monitor_event['timestamp']))).timestamp()
 
-            if not self.redis.exists(
```

Comment:
why is this check not included here?

Reply:
the check is now moved to the database file instead. it will never even reach the detection exchange if the BGP update is already handled